code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm

```diff
@@ -297,8 +297,8 @@
 	result = /obj/item/food/popsicle/jumbo
 	subcategory = CAT_ICE
 
-/datum/crafting_recipe/food/nogga_black
-	name = "Nogga black"
+/datum/crafting_recipe/food/licorice_creamsicle
+	name = "Licorice popsicle"
```

Comment:
Is this intentional?

Reply:
whoops, forgot to rename to Void Bar :tm: there